webapp/api/marketo.py

```diff
@@ -84,7 +84,11 @@ def get_user(self, email):
         response = self.request(
             method="GET", url=LEAD_BY_EMAIL, url_args=dict(email=email)
         )
-        return response["result"][0]
+        return (
+            response["result"][0]
+            if "result" in response and len(response["result"]) > 0
+            else {}
```

Comment:
As @jkfran mentioned I think this can be None rather than an empty dict.

Might also be good to update the same kind of return on ~ line 99?

Reply:
Yeah, IMO it should return `None`, it makes it clear that there is no user. I also prefer the basic if in this case just because the condition is too long. Like:
```python
if "result" in response and len(response["result"]):
    return response["result"][0]
else:
    return None
```